compiler/rustc_ast_passes/src/ast_validation.rs

```diff
@@ -710,10 +710,12 @@ impl<'a> AstValidator<'a> {
         }
     }
 
-    fn maybe_lint_missing_abi(&mut self, span: Span, id: NodeId) {
+    fn check_missing_abi(&mut self, span: Span, id: NodeId) {
```

Comment:
The check has already happened before we get here, so `check_` isn't really right.  Let's go with `handle_missing_abi` unless someone has a better idea.